css/style.css

```diff
@@ -51,41 +40,84 @@ p {
 
 }
 
-.header-logo { 
-    margin: 0 auto;
+.header__logo { 
+    display: flex;
+    align-items: center;
 }
 
-.header-text {
+.header__links-group {
     display: flex;  
     gap: 70px;
+    align-items: center;
+    height: 100%;
 }
 
-.header-text-link {
+.header__link {
```

Comment:
in the original mockup they have no underline for the links ![image](https://user-images.githubusercontent.com/9587715/203160293-7e142615-2446-4a99-987c-e97237343fe3.png)
your: 
![image](https://user-images.githubusercontent.com/9587715/203160447-58d453b0-2b21-46a8-af4b-6f4520340718.png)